conf/modules.config

```diff
@@ -257,7 +257,7 @@ process {
     }
 
     withName: '.*:FEATURECOUNTS_CDS' {
-        ext.args = params.orf_caller == 'transdecoder' ? '-g ID -t CDS -F gff3' : '-g ID -t CDS -F gtf'
+        ext.args = '-g ID -t CDS -F gtf'
```

Comment:
Does this work for all three orf callers? It looks like we had a non-functioning lambda before (should have `{}` around it, I think). Don't we need the test anymore, transdecoder outputs the same type of gff as the other two?

Reply:
Yes, I think the issue wasn't the annotation file but a step before. I tested all three orf callers and they work now.